demo/index.html

```diff
@@ -454,23 +454,44 @@
     }());
   </script>
 
-  <p>Here's an image of the line chart:</p>
-
-  <div id='line-chart-img-container'></div>
-
-  <script>
-    (function() {
-      var chart = document.getElementById('line-chart');
-      var imgContainer = document.getElementById('line-chart-img-container');
-
-      chart.addEventListener('google-chart-render', function() {
-        var img = document.createElement('img');
-        img.src = chart.getImageURI();
-        imgContainer.innerHTML = '';
-        imgContainer.appendChild(img);
+  <p>This demonstrates the ways to get the chart image URI:</p>
+
+  <dom-module id="my-el">
```

Comment:
defining elements in the main doc need to use `whenReady`

https://www.polymer-project.org/1.0/docs/devguide/registering-elements.html#main-document-definitions